x-pack/legacy/plugins/monitoring/server/lib/metrics/elasticsearch/metrics.js

```diff
@@ -62,7 +62,7 @@ const nodeCpuUtilizationLabel = i18n.translate('xpack.monitoring.metrics.esNode.
   defaultMessage: 'CPU Utilization'
 });
 const nodeGcCount = i18n.translate('xpack.monitoring.metrics.esNode.gsCountTitle', {
-  defaultMessage: 'GC Count'
+  defaultMessage: 'GC Events (per sec)'
```

Comment:
I think this should say `GC rate` and in a separate PR, investigate adding the units to the label for all rate graphs (since I don't think we have them in any of them). WDYT?

Reply:
I'm fine with that.

I'll do two things, then:

1) Update this PR per your suggestion.
2) Open a new GH issue to track the modification of existing rate graphs which need unit information added.

Reply:
@chrisronline  Issue filed here: https://github.com/elastic/kibana/issues/41458

I left it a bit vague though I have some ideas on implementation that we could discuss once we get there.